cmd/image/upload/upload.go

```diff
@@ -204,6 +217,7 @@ func init() {
 	Cmd.Flags().StringVarP(&pkg.ImageCMDOptions.ImageName, "file", "f", "", "The PATH to the file to upload.")
 	Cmd.Flags().StringVarP(&pkg.ImageCMDOptions.ObjectName, "cos-object-name", "o", "", "Cloud Object Storage Object Name(Default: filename from --file|-f option)")
 	Cmd.Flags().StringVarP(&pkg.ImageCMDOptions.Region, "bucket-region", "r", "us-south", "Cloud Object Storage bucket region.")
+	Cmd.PersistentFlags().BoolVar(&pkg.ImageCMDOptions.E2e, "e2e", false, "option for running image upload e2e tests")
```

Comment:
e2e flag to these command looks a little vague, wondering if you can use `no-prompt` itself here and handle the things accordingly.